src/redux/books/booksActions.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] [REQUIRED] Kindly fix linter errors instead of disabling them. This is important in keeping high code standards.